README.md

```diff
@@ -0,0 +1,113 @@
+# pmemfile
+
+Pmemfile project's goal is to provide low-overhead userspace implementation of
+file APIs using persistent memory.
+It consists of 2 major compoments:
+- libpmemfile-posix - provides syscall-like API which can be used by applications
+- libpmemfile - provides transparent access to libpmemfile-posix pools
+
+# How to build #
+
+Requirements:
+- cmake >= 3.3 (XXX: can we build with older versions?)
```

Comment:
I think we should still be aware of what versions are actually available (RHEL usually has pretty old version of anything). Otherwise, I would go for cmake 3.7 and C11 instead of C99, but we can't apparently.